src/local-history/local-history-manager.ts

```diff
@@ -5,6 +5,8 @@ import { LocalHistoryPreferencesService } from './local-history-preferences-serv
 import * as os from 'os';
 import * as crypto from 'crypto';
 import * as moment from 'moment';
+import * as minimatch from 'minimatch';
```

Comment:
I believe `minimatch` should be added to the `package.json`.
At the moment it is being found due to a nested dependency.